source/acquaman/AMEXAFSScanActionControllerAssembler.cpp

```diff
@@ -213,10 +213,12 @@ AMAction3 *AMEXAFSScanActionControllerAssembler::generateActionTreeForEXAFSStepA
 
 			for(int x = 0; x < detectors_->count(); x++){
 
-				AMAction3 *detectorSetDwellAction = detectors_->at(x)->createSetAcquisitionTimeAction(variableIntegrationTimes.at(i));
+				if(detectors_->at(x)->supportsSynchronizedDwell()) {
```

Comment:
I agree with this comment.  The supportsSynchronizedDwell() may not be the call you want because I believe that is specifically referencing the Synchronized dwell time that we have replaced and I've even made an issue to remove from the AMDetector API (re: #920).